main.go

```diff
@@ -108,6 +109,26 @@ func main() {
 		return
 	}
 
+	scaledObjectMaxReconciles := 10
+	scaledObjectMaxReconcilesStr := os.Getenv("KEDA_SCALEDOBJECT_CTRL_MAX_RECONCILES")
+	if scaledObjectMaxReconcilesStr != "" {
+		scaledObjectMaxReconciles, err = strconv.Atoi(scaledObjectMaxReconcilesStr)
+		if err != nil {
+			setupLog.Error(err, "Invalid KEDA_SCALEDOBJECT_CTRL_MAX_RECONCILES")
+			return
+		}
+	}
```

Comment:
Is it worth still falling back to the default if the env var exists but is invalid?

either way, code like this appears in a few different places, just with different env var names and defaults. maybe pull it out to a function?

Reply:
I would prefer failing fast if the env variable is invalid, to let users know that have done a mistake. It might give the wrong impression about the behaviour, 

Yeah, pulling it out to a separate function is a good idea. I'll do that